Inference/c++/ernie/mkldnn/README.md

```diff
@@ -117,7 +115,7 @@ export KMP_BLOCKTIME=1
 
 |     Model    |  FP32 Accuracy | QAT INT8 Accuracy | Accuracy Diff |
 |:------------:|:----------------------:|:----------------------:|:---------:|
-|   Ernie      |          79.76%        |         79.28%         |   -0.48%  |               
+|   Ernie      |          79.76%        |         79.96%         |   +0.20%  |               
```

Comment:
it is `+0.25%`

Reply:
Please take a look at slack, you sent me
```
FP32: avg accuracy: 0.797590
INT8: avg accuracy: 0.799598
which gives +0.25%
```
but why the difference is 0.25? I use google to calculate, it is 0.002008, so +0.2008%

Do you mean compared to origianl fp32 model, not qat fp32 model, in this way, we should do another accuracy check for origianl fp32 model

Reply:
If we are writing about the % increase from FP32 accuracy to INT8 accuracy the formula is:
`(acc_INT8 - acc_FP32)/acc_FP32`, so `(0.799598 - 0.797590)/0.797590 = 0.0025`, i.e. `+0.25%`.

Reply:
Ah, I got what you meant. You are writing about the absolute difference in accuracy, not about the % increase. Then that is fine, `+0.20%` is correct. Thank you!

Reply:
Solved